### PR TITLE
tours guide command

### PR DESCRIPTION
**Pull request by spekss**
command with useful links and list of official smogon tournament

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
+        toursguide: function (target, room, user) {
```
Comment (Bryan-0) [inline commands.js]: You're using spaces instead of TAB, change it. Use TAB instead of spaces.

File: commands.js [hunk 2]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
...
+  		if (!this.canBroadcast()) return;
```
Comment (Bryan-0) [inline commands.js]: You're mixing spaces & tabs, only use tabs.

Also, there is a trailing space in the last part of ``return; `` remove it.

File: commands.js [hunk 3]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
+        toursguide: function (target, room, user) {
+  		if (!this.canBroadcast()) return; 
+  		this.sendReplyBox(
```
Comment (Bryan-0) [inline commands.js]: Here too, mixed spaces & tabs.

File: commands.js [hunk 4]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
...
+			'<img src="http://www.smogon.com/media/forums/images/badges/worldcup.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/world-cup-of-pokemon.234/">Worl Cup of Pokemon</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/grandslam.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-grand-slam.208/">Smogon Grand Slam</a><br>' +
+			'<img src="http://imgur.com/nIdW9QW.png" width="16" height="16"><a herf="http://www.smogon.com/forums/forums/the-smogon-classic.334/">The Smogon Classic</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/olt.png" width="16" height="16"><a href="http://www.smogon.com/forums/threads/smogons-official-ladder-tournament-iii-cycle-4-signups-read-post-178.3580529/#178">Smogons Official Ladder Tournament (cycle #4)</a><br>' +
+			'<img src="http://www.smogon.com/media/community/awards/ost.png" width="16" height="16"><a href="http://www.smogon.com/forums/threads/smogon-tournament-12-signups.3562131/">Smogon Official Smogon Tournament</a>'
+		); 
+	},
```
Comment (Bryan-0) [inline commands.js]: There's a trailing space in line 3011 ``);``.

Line 3012: At the beginning you use a space & then tab, remove that space and just use tab.

File: commands.js [hunk 5]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
...
+  		this.sendReplyBox(
+			'<b>Tournament Rules and General Guidelines</b><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/">List of TD, rules, and more explication.</a><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/#post-6823208">Definition of Ghosting</a><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/#post-6703482">Guide to Scheduling</a><br>' +
+			'<hr>' +
+			'<b>List of Official Smogon Tournaments</b><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/tour.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-tour.49/">Smogon Tour</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/spl.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-premier-league.130/">Smogon Premier League</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/worldcup.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/world-cup-of-pokemon.234/">Worl Cup of Pokemon</a><br>' +
```
Comment (CrestfallPS) [inline commands.js]: Typo, should be "World"

**Comment by spekss** [comment]
okey , thank you very much  Bryan-0  and  CrestfallPS !

File: commands.js [hunk 6]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
+	toursguide: function (target, room, user) {
+		if (!this.canBroadcast()) return;
```
Comment (Slayer95) [inline commands.js]: ``runBroadcast()``

**Comment by Zarel** [comment]
Do we actually want this, though? @sirDonovan?

**Comment by sirDonovan** [comment]
In my opinion, this isn't necessary on main.

**Comment by spekss** [comment]
I think the command can be useful on strategic room, 
If a person does not know or find a tournament or the rules , the staff just has to use the command

**Comment by panpawn** [comment]
If this does get added, it should probably go in ``chat-plugins/info.js`` imo.

Also, use ``<br />`` instead of ``<br>``

File: commands.js [hunk 7]
```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
```
Comment (AustinXII) [inline commands.js]: I don't think anyone would use tourexplication...

**Comment by spekss** [comment]
done panpawn

File: commands.js (hunk 7 above)
Comment (AustinXII) [inline commands.js]: Or tourexplination

**Pull request closed without merging**